scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Maybe we could stick with `build/repository.sh` in alignment with `deploy/repository.sh`? It's difficult for me to find the right name related to functionality since this script apart from providing version info also builds package and prepares `latest.txt` pointer to the most recent version.